Frontend/ui-common/src/Engineering/Helpers/Utils.js

```diff
@@ -78,12 +81,12 @@ export const getWeeksInMonth = function (year) {
         let week = [];
         for(var i = 0 ; i < 7; i++){
           if(startPadding){
-            let obj = { utcDate: "", date: 0, shortMonth: "", year: year, intMonth: month,
+            let obj = { day: "", date: 0, shortMonth: "", year: year, intMonth: month,
                         isInRange : false, isStart: false , isEnd: false }
             week.push(obj);
             startPadding --;
           }else{
-            let obj = { utcDate: date.toISOString(), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: year, intMonth: month,
+            let obj = { day: getISODate(date), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: year, intMonth: month,
```

Comment:
Please rename this `obj` to something something descriptive

Reply:
done